repspark_at/actions/actions_login_page.py

```diff
@@ -11,24 +10,28 @@ class LoginPage(BasePage):
     def set_user_name(self, username):
         email_field = LoginPageElements.email_field(self)
         self.assertTrue(email_field.is_displayed())
+        #assert True, email_field.is_displayed
         email_field.click()
         email_field.send_keys(username)
 
     def set_password(self, password):
         password_field = LoginPageElements.password_field(self)
-        self.assertTrue(password_field.is_displayed())
+        #self.assertTrue(password_field.is_displayed())
+        #assert password_field.is_displayed(), "password field isn't displayed"
         password_field.click()
         password_field.send_keys(password)
 
     def click_login_button(self):
         login_button = LoginPageElements.login_button(self)
-        self.assertTrue(login_button.is_displayed())
+        #self.assertTrue(login_button.is_displayed())
+        assert True, login_button.is_displayed
```

Comment:
This is the incorrect assert. Please update or remove it in this pull request.

Reply:
@extsoft 
I changed all asserts in repspark_at/actions/actions_login_page.py file. Other asserts will be changed in next pull request